.github/workflows/test.yml

```
@@ -0,0 +1,35 @@
+name: Test
+
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["pypy3.10", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

Python 3.13 has been released.

```suggestion
        python-version: ["pypy3.10", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```